### PR TITLE
No forbiddenChars check for URI param idPattern

### PR DESCRIPTION
**Pull request by kzangeli**
Removed the check of forbidden characters for the value of the URI param **idPattern**

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -28,3 +28,4 @@ Add:  PUT /v2/entities/<entityId>/attrs/<attrName> (Issue #991)
 Add:  GET /v2/types/<type> (Issue #1028)
 Add:  APPEND_STRICT action type for POST /v1/updateContext operation
 Add:  REPLACE action type for POST /v1/updateContext operation
+Fix:  Removed the check of forbidden characters for the values of the URI parameter 'idPattern'
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: Is there any issue number associated to this? (It is ok if there isn't: just checking).

Comment (kzangeli) [inline CHANGES_NEXT_RELEASE]: I didn't create any issue. It's pretty straightforward and I didn't see the need for further explanation.

Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: NTC

**Comment by fgalan** [comment]
LGTM

**Pull request merged**